### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by doncem**

### CODE REVIEW

File: microservices-schema.json [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+                            "default": "",
+                            "examples": [
+                                "IdAM"
+                            ]
+                        },
+                        "hard": {
+                            "type": "boolean",
+                            "$id": "https://raw.githubusercontent.com/hmcts/reform-api-docs/master/docs/microservices.json/items/properties/dependencies/items/properties/hard",
+                            "title": "???",
+                            "description": "???",
```
Comment (rkondratowicz) [inline microservices-schema.json]: provide title and description

File: microservices-schema.json [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+                                true
+                            ]
+                        },
+                        "apis": {
+                            "type": "array",
+                            "$id": "https://raw.githubusercontent.com/hmcts/reform-api-docs/master/docs/microservices.json/items/properties/dependencies/items/properties/apis",
+                            "items": {
+                                "$id": "grizzlybear",
+                                "title": "???",
+                                "description": "???"
```
Comment (rkondratowicz) [inline microservices-schema.json]: provide title and description

File: microservices-schema.json [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+                            "default": false,
+                            "examples": [
+                                true
+                            ]
+                        },
+                        "apis": {
+                            "type": "array",
+                            "$id": "https://raw.githubusercontent.com/hmcts/reform-api-docs/master/docs/microservices.json/items/properties/dependencies/items/properties/apis",
+                            "items": {
+                                "$id": "grizzlybear",
```
Comment (rkondratowicz) [inline microservices-schema.json]: grizzlybear?

File: microservices-schema.json [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+                    }
+                }
+            },
+            "apis": {
+                "type": "array",
+                "$id": "https://raw.githubusercontent.com/hmcts/reform-api-docs/master/docs/microservices.json/items/properties/apis",
+                "items": {
+                    "$id": "sunbear",
+                    "title": "???",
+                    "description": "???"
```
Comment (rkondratowicz) [inline microservices-schema.json]: provide title and description

File: microservices-schema.json [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+                            }
+                        }
+                    }
+                }
+            },
+            "apis": {
+                "type": "array",
+                "$id": "https://raw.githubusercontent.com/hmcts/reform-api-docs/master/docs/microservices.json/items/properties/apis",
+                "items": {
+                    "$id": "sunbear",
```
Comment (rkondratowicz) [inline microservices-schema.json]: sunbear?

File: microservices-schema.json [hunk 6]
```diff
@@ -0,0 +1,120 @@
...
+                    "title": "???",
+                    "description": "???"
+                }
+            },
+            "version": {
+                "type": "string",
+                "$id": "https://raw.githubusercontent.com/hmcts/reform-api-docs/master/docs/microservices.json/items/properties/version",
```
Comment (rkondratowicz) [inline microservices-schema.json]: are those links supposed to work?

File: microservices-schema.json [hunk 7]
```diff
@@ -0,0 +1,120 @@
...
+                "description": "Link to API specification json file",
+                "default": null,
+                "examples": [
+                    "https://hmcts.github.io/reform-api-docs/specs/payments.json"
+                ]
+            },
+            "colour": {
+                "type": "string",
+                "$id": "https://raw.githubusercontent.com/hmcts/reform-api-docs/master/docs/microservices.json/items/properties/colour",
+                "title": "Network colout",
```
Comment (rkondratowicz) [inline microservices-schema.json]: typo in 'colout'

File: microservices-schema.json (hunk 1 above)
Comment (doncem) [reply microservices-schema.json]: I feel like schema is a bit too big. I don't know what certain fields suppose to mean - do they have any value at all, but basically I took the existing data blob and generated schema for it. Therefore certain values I do not know how to fill in and leave that to @jasonpaige.

File: microservices-schema.json (hunk 3 above)
Comment (doncem) [reply microservices-schema.json]: Schema generator populates when there is no information about the object (array items in this case)

File: microservices-schema.json (hunk 6 above)
Comment (doncem) [reply microservices-schema.json]: Schema autogenerates

File: microservices-schema.json (hunk 7 above)
Comment (doncem) [reply microservices-schema.json]: 😇

**Pull request merged**